x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/aggregate/TopTests.java

```diff
@@ -274,7 +274,7 @@ public static Iterable<Object[]> parameters() {
                             new TestCaseSupplier.TypedData(1, DataType.INTEGER, "limit").forceLiteral(),
                             new TestCaseSupplier.TypedData(null, DataType.KEYWORD, "order").forceLiteral()
                         ),
-                        "third argument of [] cannot be null, received [order]"
+                        "third argument of [source] cannot be null, received [order]"
```

Comment:
I wonder if `source` should be `TOP` or something. `source` feels sort of surprising here even though it's easy to make it the test.

Reply:
You could call `functionName()` to get the function I think.

Reply:
It's currently a static Source. Making it dynamic is possible, but requires also changing the Configuration.
As commented in other thread here, I'll try that later, but it requires some extra refactor, and I feel like it's better in a separate PR.
I thoguht about changing that "source" to something more localizable like "$source$" or even "function()". But at this point, I'm not sure it adds much